gems/quilt_rails/README.md

```diff
@@ -6,13 +6,20 @@ This document focuses on Rails integration. For details of `@shopify/react-serve
 
 ## Quick Start
 
-Create a Rails project using `dev init` then:
+### Add Ruby dependencies
+
+`bundle add sewing_kit quilt_rails`
+
+First, create a Rails project using `dev init`. Next, run `bundle exec rake quilt:install`. This will install the Node dependencies, provide a basic React app (in TypeScript) and mount the Quilt engine inside of your `config/routes.rb` file.
```

Comment:
should we add a super-simple `sewing_kit:install` to `sewing_kit` that poops out a mostly empty `sewing-kit.config.ts`?